.github/workflows/pr-check.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v4
         with:
-          fetch-depth: '0'
```

Comment:
This deletion is the only part I wasn't sure about - think it is safe to replace fetch entire history with (smaller) fetch of all tags.
There is no way to fetch just the latest tag (which would have been sufficient)